src/platform/qt/library/LibraryController.h

```diff
@@ -65,8 +65,12 @@ class AbstractGameList {
 	virtual void addEntry(const LibraryEntry&);
 	virtual void updateEntry(const LibraryEntry&);
 	virtual void removeEntry(const QString&);
+	virtual void setShowFilenameInLibrary(bool newValue);
 
 	virtual QWidget* widget() = 0;
+protected:
+	bool showFilenameInLibrary = false;
```

Comment:
```suggestion
	bool m_showFilename = false;
```

Is this actually used anywhere? And why protected?